src/analysis/ODE/gronwall.lean

```diff
@@ -0,0 +1,282 @@
+/-
+Copyright (c) 2020 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import analysis.calculus.mean_value analysis.complex.exponential
+
+/-!
+# Grönwall's inequality
+
+The main technical result of this file is the Grönwall-like inequality
+`norm_le_gronwall_bound_of_norm_deriv_right_le`. It states that if `f : ℝ → E` satisfies `∥f a∥ ≤ δ`
+and `∀ x ∈ [a, b), ∥f' x∥ ≤ K * ∥f x∥ + ε`, then for all `x ∈ [a, b]` we have `∥f x∥ ≤ δ * exp (K *
+x) + (ε / K) * (exp (K * x) - 1)`. This inequality is a generalization of a special case of
```

Comment:
You write "This inequality is a generalization of a special case of Grönwall's inequality". I don't think there is a unique canonical form of Grönwall's inequality, so the next sentence (saying the differences with the usual Grönwall) is not really understandable. I would suppress these two sentences (from "This inequality" to " an additive constant `ε`."), as saying your inequality is Grönwall-like looks perfectly clear to me.

Reply:
Done.